templates/blog/post_detail.html

```diff
@@ -0,0 +1,30 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <div class="post">
+    <h2 class="text-center">{{ post.title }}</h2>
+    <hr style="height: 2px; background-color: #9D7753">
```

Comment:
Inline styles are generally not recommended. It's better to move this style to your CSS file and use a class to apply it. This would make your code cleaner and more maintainable.